### PR TITLE
Add timestamp to the JSON output

### PR DESCRIPTION
**Pull request by gnud**
- Updated readme to include batch example
- The timestamp is only included via json argument and verbose all together, providing ability to monitor the results via some tool

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -28,6 +28,11 @@ $ speed-test --help
     --verbose -v  Output more detailed information
 ```
 
+## Batch monitoring
```
Comment (sindresorhus) [inline readme.md:31]:
```suggestion
## Tips

### Batch monitoring
```

File: readme.md [hunk 2]
```diff
@@ -28,6 +28,11 @@ $ speed-test --help
...
+
+```bash
+while true; do sleep 30; node cli.js -v -j; done >> ~/net.log
```
Comment (sindresorhus) [inline readme.md:34]: Use the long flags.

**Review by sindresorhus — CHANGES_REQUESTED** [review]

**Comment by sindresorhus** [comment]
Can you mention in the readme that the JSON output also includes a timestamp?

**Review by gnud — COMMENTED** [review]
Added suggested changes, please confirm if they were applied.

**Comment by sindresorhus** [comment]
> please confirm if they were applied.

I don't understand what this means. You can confirm yourself.

**Comment by sindresorhus** [comment]
You still need to do https://github.com/sindresorhus/speed-test/pull/67#issuecomment-717872597

**Review by gnud — COMMENTED** [review]
All done :).

File: readme.md [hunk 3]
```diff
@@ -28,6 +28,15 @@ $ speed-test --help
...
 
+## Tips
+
+### Batch monitoring
+
+```bash
+while true; do sleep 30; node cli.js --verbose --json; done >> ~/net.log
+```
+
+**Note**: using the **"--verbose"** and **"--json"** will include a timestamp **e.g**: "1603971242534" property generated now in form of a timestamp, good for recording the metrics in some monitoring software.
```
Comment (sindresorhus) [inline readme.md:39]: This should be documented generally. It's not specific to "batch monitoring".

**Comment by sindresorhus** [comment]
@gnud Friendly bump :)

**Comment by sindresorhus** [comment]
Closing for lack of activity.

**Pull request closed without merging**